templates/v1/basic.py

```diff
@@ -54,25 +53,6 @@ def parse_arguments():
     return parser.parse_args()
 
 
-def main():
```

Comment:
I'm not a fan of this being removed. The template should make obvious what's going on. That's kind of the point of the template. So removing the strategy and agent removes the point of the template in my opinion.

Reply:
Right, I'll restore the previous version